biogtr/config.py

```diff
@@ -13,6 +13,7 @@
 from biogtr.datasets.microscopy_dataset import MicroscopyDataset
 from omegaconf import DictConfig, OmegaConf
 from typing import Union, Iterable
+from pprint import pprint
```

Comment:
Maybe good to think about verbosity levels?

Reply:
can prob do this when we release for general use